typings-examples.md

```diff
@@ -0,0 +1,92 @@
+# Examples
+Here are some examples you can model after when writing your own typings.
+
+## Single function
+
+Exposing a single function.
+
+```ts
+declare function domready (...): any;
+
+export = domready
+```
+
+* https://github.com/typed-typings/npm-domready
+
+Function with overloads
+
+```ts
+declare function xtend <A> (a: A): A;
+declare function xtend <A, B> (a: A, b: B): A & B;
+export = extend;
```

Comment:
The name here should be "xtend" ? (or change the name in the two previous lines)